src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -124,6 +124,11 @@ public enum ProtectionLevel
         public static bool operator !=(System.Net.Security.SslApplicationProtocol left, System.Net.Security.SslApplicationProtocol right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public sealed partial class SslStreamCertificateContext
+    {
+        internal SslStreamCertificateContext(System.Security.Cryptography.X509Certificates.X509Certificate2 target, System.Security.Cryptography.X509Certificates.X509Certificate2[]? intermediates = null) { throw null; }
```

Comment:
Was this auto-generated?  I'd expect it to just say `internal SslStreamCertificateContext();`, since all that's important is that it hides the default public ctor.

Reply:
updated. and no, this was not generated. I just put in what the real constructor was.